src/templates/bootstrap5/file/form.ejs

```diff
@@ -119,7 +119,7 @@
         <span>
           <img ref="fileImage" src="" alt="{{file.originalName || file.name}}" style="width:{{ctx.component.imageSize}}px">
           {% if (!ctx.disabled) { %}
-            <i tabindex="0" class="{{ctx.iconClass('remove')}}" ref="removeLink"></i>
+            <i tabindex="0" class="{{ctx.iconClass('remove')}}" ref="removeLink" aria-label="{{ctx.t('Remove button. Press to remove ' + (file.originalName || file.name) + ' file')}}"></i>
```

Comment:
```suggestion
            <i tabindex="0" class="{{ctx.iconClass('remove')}}" ref="removeLink" aria-label="{{ctx.t('Remove button. Press to remove {{fileName}}', { fileName: file.originalName || file.name })}}"></i>
```

Reply:
@brendanbond I'm not sure we can use nested params inside of {{ }}, e.g. `{{ ctx.t(' {{ fileName }} ',  { fileName: foo }) }}` is not supported by ejs templates